### ISSUE TITLE
Special characters are not being recognised causing files to get corrupted and redownloaded instead of checking

### ISSUE THREAD

**Issue by Nowena** [transmission/transmission#6410]
### What is the issue?

I have tens of torrents paused and not seeding due to this behaviour.

"Special characters" such as ‘ ’ – { } and a large strings of numbers cause this. Sometimes several of these characters are present on a torrent name at the same time. Pretty sure some random ‘ ’ in place of ' ' as part of a file name are causing an entire torrent to not get recognised/checked at all. And even a simple space "year -  name/etc" as part of a torrent name seems to obfuscate the checking process and these torrent are never checked/recognised.

Therefore, my files get all corrupted and start redownloading all content. I have over 50 torrents or so with this predicament. Some large. The amount of buffer consumed on redownloading them to just be able to seed them again is not really an option.

I can't even seem to be able to rename the problematic torrents and continue seeding. The integrity of all my files, I assure is 100% healthy. They just get corrupted if I do not pause them right away or untick "start after adding".

Is there anything I can do regarding this from my part? Some kind of fickle windows configuration/tweaking/tampering to correct this?

Or is it not really on my end and it is an issue caused by the Transmission client itself? I have been using Transmission for quite a while and none of the subsequent updates correct this.

### Which application of Transmission?

Qt app on Windows

### Which version of Transmission?

4.0.5

**Comment by killemov** [comment]
Some torrent creators keep on using illegal Windows filename characters in their torrents. You could rename the files both in Transmission and on the filesystem. But having Transmission replace the illegal characters with an _ , for example, automatically would be preferable.

**Comment by Nowena** [comment]
Yes, they do. And it is annoying they keep on doing this. Although I would not call these "illegal characters".

They can be input anywhere on windows unlike ? / etc.

However, this wasn't an issue on neither utorrent nor deluge nor qtbittorrent.

But I am never coming back to those. They are simply not good. Transmission meets my needs.

I tried renaming torrents and they still would not seed. That was a previous Transmission version. Will try that again and automatically replacing those characters.

Yes, preferable to manually renaming every torrent with this issue to something else.

Still, this appears to be a thing with Transmission itself.

**Comment by Nowena** [comment]
I just tried, renaming still does not let me seed anything. The torrents are never recognised and they start redownloading. And I did not see any option to automatically replace "problematic" characters anywhere.

May you give details on how can I achieve this?

**Comment by tearfur** [comment]
I *think* what you are seeing is a variant of #1700 and #4693.

This should be fixed in the latest [nightlies](https://build.transmissionbt.com/job/trunk-win32/), but the fix will not be in `4.0.x` versions. Please help test and report back the results if you're willing to.

**Comment by Nowena** [comment]
All of these reports seem to be the same symptom. What if someone doesn't feel like using nightlies? I don't. Many others also don't. There should be a reason for this not being fixed in 4.0.x/standard versions and I can't imagine what it is.

**Comment by tearfur** [comment]
`4.1.0` released.
